README-dnszone.md

```diff
@@ -172,7 +173,7 @@ Variable | Description | Required
 `name_server`| Authoritative nameserver domain name | no
 `admin_email`| Administrator e-mail address | no
 `update_policy`| BIND update policy | no
-`dynamic_update`| Allow dynamic updates | no
+`dynamic_update` \| `dynamicupdate` | Allow dynamic updates | no
```

Comment:
Add this change to the plugins/ipadnszone.py file (under DOCUMENTATION).

Reply:
Done.